src/govuk-helpers.js

```diff
@@ -0,0 +1,14 @@
+const lookupError = (name, errors) => {
+  const error = errors && errors.find(s => s.targetName === name)
+  return error && error.text
+}
+
+export const lookupMeta = (name, errors) => {
+  if (typeof errors === 'string') return {}
+  const error = lookupError(name, errors)
+  return error ? { touched: Boolean(error), error } : {}
+}
+
+export const onHandleErrorClick = targetName => {
+  document.getElementsByName(targetName)[0].focus()
```

Comment:
note to @JamieDraperUK I have used `focus` instead of the `scrollIntoView` that you used.  Can't see why you wouldn't use `focus` and GDS wants it to be focus.  If that makes sense then I'll raise a PR for the govuk-react stuff to get them suggest `focus` instead